bitnami/thanos/templates/querier/deployment.yaml

```diff
@@ -62,7 +62,7 @@ spec:
             - --store.sd-files=/conf/servicediscovery.yml
             {{- end }}
             {{- if and .Values.querier.dnsDiscovery.enabled .Values.querier.dnsDiscovery.sidecarsService .Values.querier.dnsDiscovery.sidecarsNamespace }}
-            - --store=dnssrv+_grpc._tcp.{{ .Values.querier.dnsDiscovery.sidecarsService }}.{{ .Values.querier.dnsDiscovery.sidecarsNamespace }}.svc.{{ .Values.clusterDomain }}
+            - --store=dnssrv+_grpc._tcp.{{ tpl .Values.querier.dnsDiscovery.sidecarsService . }}.{{ tpl .Values.querier.dnsDiscovery.sidecarsNamespace . }}.svc.{{ .Values.clusterDomain }}
```

Comment:
could you follow the same approach as we follow on https://github.com/bitnami/charts/blob/master/bitnami/thanos/values.yaml#L256 using https://github.com/bitnami/charts/blob/master/bitnami/thanos/templates/_helpers.tpl#L328

Reply:
ok, i will use that template function, thanks for guidance ;)